subsys/mgmt/mcumgr/smp.c

```diff
@@ -8,11 +8,11 @@
 #include <zephyr/kernel.h>
 #include <zephyr/device.h>
 #include <zephyr/net/buf.h>
-#include <zephyr/mgmt/mcumgr/buf.h>
-#include "mgmt/mgmt.h"
-#include "smp/smp.h"
+#include <zcbor_common.h>
```

Comment:
Don't think zcbor header is needed here, can't see any usage of zcbor structs in this file

Reply:
It is no longer there.